### DISCUSSION TITLE
i am new to the paly framework

### DISCUSSION THREAD

**Discussion by janani-reddy9** [playframework/playframework#11815] [General]
i am beginner**** please help me .

**Comment by ihostage** [comment]
For getting the help, you need to write your question 🤷‍♂️ 
For that, please edit subject and body of this discussion 😉

**Comment by janani-reddy9** [comment]
Hey, Hi.
Sorry for the message. It was accidental, I am trying to get started with. Let me post a discussion if I need to know something.
Thanks for the reply.
